ant/org.eclipse.ant.launching/remote/org/eclipse/ant/internal/launching/remote/InternalAntRunner.java

```diff
@@ -58,6 +59,17 @@
  */
 public class InternalAntRunner {
 
+	private static final boolean IS_SECURITY_MANAGER_SUPPORTED = isSecurityManagerAllowed();
+
+	private static boolean isSecurityManagerAllowed() {
+		String sm = System.getProperty("java.security.manager"); //$NON-NLS-1$
+		if (sm == null) { // default is 'disallow' since 18 and was 'allow' before
+			return !JavaEnvUtils.isAtLeastJavaVersion("18"); //$NON-NLS-1$
```

Comment:
If we require Java 21 now it might be obsolete?!

Reply:
These tests are failing when running with Java 21.  Just because the IDE requires Java 21 doesn't mean an ant task requires Java 21 or even the Eclipse ant bundles for that matter. 

Anyway, it's a copy of this already-committed code:

https://github.com/eclipse-platform/eclipse.platform/blob/9636c7e790426c77388df64dc5f83ca37222ac86/ant/org.eclipse.ant.core/src_ant/org/eclipse/ant/internal/core/ant/InternalAntRunner.java#L86-L95